### PR TITLE
Restructure data directories

### LINKED ISSUE #609: Reorganise current News articles data files
**Is your feature request related to a problem? Please describe.**
Currently, News test article data is in the `/data` directory, with News production article data in the `/data/prod` directory. This does not help future work, with multiple services, e.g. Persian.

**Describe the solution you'd like**
The News test and prod article should be moved to these directories:
~/data/news/test~ now `/data/test/news`
~/data/news/prod~ now `/data/prod/news`
And the onward journeys directory should be moved to
~/data/news/test/onward-journeys~ `data/test/news/onward-journeys`

The data validator npm scripts should run against the files in the new locations.

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #609

- This PR restructures the `data/` directory to allow for multiple services (news and persian). 
- It adds a nested `test` directory to clearly seperate what is test/prod content. 
- Replaces data paths such as `/data/c0000000001o.json` with `/data/news/test/c0000000001o.json`.
- Changes the dataValidator logic to recursive through all the directories nested within `/data`.
- Implements logic to ignore the `onward-journeys/` directory as the schema has not yet been defined.
- Increase the expected number of scenarios read from 29 to 41. Previously their were 29 homed directly under `/data` by adding prod and onward journeys this number becomes 41.

- [x] Tests added for new features
- ~[ ] Test engineer approval~

### CODE REVIEW

File: src/app/dataValidator/helpers/dataLoader/readScenario.js [hunk 1]
```diff
@@ -2,21 +2,36 @@ const fs = require('fs');
 const path = require('path');
 const { validateData } = require('../validators/validateData');
 const { countScenarios } = require('../../utilities/countScenarios');
+const readdirSync = require('./readdirSync');
+
+const ifDirectoryValidateNestedFiles = fullFileName => {
+  const fileStats = fs.statSync(fullFileName);
+
+  if (fileStats.isDirectory()) {
+    readdirSync.readdirSync(fullFileName);
```
Comment (pjlee11) [inline src/app/dataValidator/helpers/dataLoader/readScenario.js]: This does not use destructing assignment as it creates the following error: 
```
TypeError: readdirSync is not a function
```

This is due to the recursive nature of the files.

**Review by jamesbhobbs — DISMISSED** [review]
data/prod/c0000000028o.json → data/test/persian/c0000000028o.json

This moves from prod to test. Is this expected? I think not.

**Comment by pjlee11** [comment]
@benjaminhobbs that is worth commenting on but if you also see `data/prod/c0000000028o.json` → `data/prod/persian/c0000000028o.json` it highlights that we have prod stories in the test directory also. 
NB: Both news production stories `c0000000025o.json` and `c0000000027o.json` existed in both `data/` and `data/prod/` prior to this PR

**Review by jtart — DISMISSED** [review]

**Review by jtart — APPROVED** [review]

**Review by sareh — APPROVED** [review]
👍 Looks good!

**Pull request merged**